Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferences.mm

```diff
@@ -554,15 +554,34 @@ - (_WKWebsiteModalContainerObservationPolicy)_modalContainerObservationPolicy
     return WebKit::modalContainerObservationPolicy(_websitePolicies->modalContainerObservationPolicy());
 }
 
-#if USE(APPLE_INTERNAL_SDK)
-#import <WebKitAdditions/WKWebpagePreferencesAdditions.mm>
-#else
+- (BOOL)lockdownModeEnabled
+{
+    return _websitePolicies->captivePortalModeEnabled();
+}
+
+- (void)_setLockdownModeEnabled:(BOOL)lockdownModeEnabled
```

Comment:
I know you merely moved the code but I don't get why we have a leading underscore here.

Reply:
Fixed! I'm guessing it wouldn't have worked with the underline (or at least not done what we expected).